bosh-stemcell/lib/bosh/stemcell/stage_collection.rb

```diff
@@ -14,6 +14,21 @@ def operating_system_stages
       ubuntu_os_stages
     end
 
+    def kernel_stages
+      if operating_system.variant == "fips"
+        %I[
+          system_#{infrastructure.name}_fips_kernel
+          base_fips_apt
+          system_kernel_modules
```

Comment:
Should `system_kernel_modules` be pulled out from like 27

Reply:
yes i just fixed this yesterday.